website_snippet_product_category/models/product_public_category.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2020 Tecnativa - Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ProductPublicCategory(models.Model):
+    _inherit = ["product.public.category", "website.published.mixin"]
```

Comment:
When inheriting from multiple models, you have to explicitly put `_name`
```suggestion
    _inherit = ["product.public.category", "website.published.mixin"]
    _name = "product.public.category"
```